nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -230,6 +230,12 @@ language governing permissions and limitations under the License. -->
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.mockftpserver</groupId>
```

Comment:
ASLv2

https://sourceforge.net/p/mockftpserver/code/HEAD/tree/MockFtpServer/LICENSE.txt

Also, scope is test only. I was under the impression the NOTICE and LICENSE are only required when code is bundled as part of the NAR packaging?